### DISCUSSION TITLE
Post to Layout's action instead of Page's action

### DISCUSSION THREAD

**Discussion by tomaszferens** [remix-run/remix#9007] [General]
I have following routes:

1. form.$form.tsx (page)
2. form.tsx (layout)
3. form_._index.tsx (page)

What I tried (Form was rendered in form.$form.tsx):

1. `<Form method="POST">` posts to `form_._index.tsx`
2. `<Form method="POST" action="/form">` posts to `form_._index.tsx`
3. `<Form method="POST" action="/form/*">` posts to `form.$form.tsx`

How to post to the action declared in layout (form.tsx) when inside form.$form.tsx page?

**Comment by kiliman** [comment]
You can't. If you have an index route, Remix will always send it to the action in your _index_ route because they both have the same URL.

The only way to use the action in your layout is if the form itself is rendered in your layout and you use `<Form method="post" action=".">`

If you must use the action in your layout, I would recommend moving that logic to your index route, or create a resource route that you can explicitly call.
